force-app/main/default/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -202,7 +208,6 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     get showCancelButton() {
         return this.isEdit 
             && !this.isExpiredTransaction
-            && !this.isPaymentStatusAuthorized()
```

Comment:
I don't see a reason to hide the Cancel button if an unprocessed gift has this Status... shouldn't we allow customers to undo the activation of the widget?

Reply:
The 'Don't Enter Payment Information' button and the payment method Combobox are what deactivates the widget

Reply:
Right, but I got into a situation where I could click "Edit Payment Information", and then had no way to cancel that change if I decided to keep the Payment Information the same.